master/buildbot/changes/bitbucket.py

```diff
@@ -163,7 +163,7 @@ def _processChanges(self, response):
                         comments=u'pull-request #%d: %s\n%s' % (
                             nr, title, prlink),
                         when_timestamp=datetime2epoch(updated),
-                        branch=self.branch,
+                        branch=branch,
```

Comment:
use ascii2unicode like the other properties read from the PR.
I would also change self.branch to self.branches wihch is more accurate and how other poller are doing it